velox/functions/lib/Re2Functions.h

```diff
@@ -48,12 +48,14 @@ enum class PatternKind {
 
 struct PatternMetadata {
   PatternKind patternKind;
-  // Contains the length of the fixed pattern for patterns of kind kFixed,
-  // kPrefix, and kSuffix. Contains the count of wildcard character '_' for
-  // patterns of kind kExactlyN and kAtLeastN. Contains 0 otherwise.
+  // Contains the length of the unescaped fixed pattern for patterns of kind
+  // kFixed, kPrefix, kSuffix and kSubstring. Contains the count of wildcard
+  // character '_' for patterns of kind kExactlyN and kAtLeastN. Contains 0
+  // otherwise.
   vector_size_t length;
-  // Contains the fixed pattern in patterns of kind kSubstring.
-  std::string fixedPattern = "";
+  // Contains the unescaped fixed pattern in patterns of kind kFixed, kPrefix,
+  // kSuffix and kSubstring.
+  std::string unescapedPattern = "";
```

Comment:
I wonder if we could keep the original name 'fixedPattern'. The new name causes lots of changes that make it hard to review this PR and also is confusing to me as I need to think what does 'unescaped' mean.